Applications/Utils/MeshEdit/queryMesh.cpp

```diff
@@ -63,6 +61,12 @@ int main(int argc, char *argv[])
     }
     selected_node_ids.insert(selected_node_ids.end(), nodeId_arg.getValue().begin(), nodeId_arg.getValue().end());
 
+    MeshLib::PropertyVector<int>* materialIds(nullptr);
```

Comment:
Please take a look at commit 8fa8d03c6761363a159e6cbfaf46d1a3b0031561 and judge yourself about the readability. I like Wenqings suggestion more than my previous implementation.